rsp_jupyter_extensions/handlers/query.py

```diff
@@ -92,24 +106,32 @@ async def _create_query(self, q_value: str, q_type: str) -> str:
 
     async def _create_tap_query(self, q_value: str) -> str:
         # The value should be a URL or a jobref ID
-        this_rsp = os.getenv("EXTERNAL_INSTANCE_URL", "not-an-rsp")
-        if q_value.startswith(this_rsp):
+        # A jobref is always 16 alphanumeric characters.
+        # Therefore: if it contains a slash, it's a URL
+        if q_value.find("/") != -1:
```

Comment:
I wonder if we also want to handle a case where the URL has a trailing slash like:
`https://data-dev.lsst.cloud/api/tap/async/a1xqwwmhpboi4qds/`